resources/styles/components/task/breadcrumbs.less

```diff
@@ -0,0 +1,59 @@
+.circle(@radius){
```

Comment:
Can you prefix the mixin with the name of the component so the names do not conflict with mixins elsewhere?